app/assets/stylesheets/responsive.scss

```diff
@@ -29,6 +29,48 @@
     padding: 10px 10px 0 10px;
     border-bottom: 0;
   }
+
+  // Responsive table behvavior inspired by
+  // https://css-tricks.com/responsive-data-tables/
+  .js--responsive_table {
```

Comment:
Actually, I've just decided I had an opinion. Let's keep them as one style on the grounds that you don't want one without the other ever since the behavior depends on both.